Kiosk/SettingsHelper.cs

```diff
@@ -622,6 +631,17 @@ public string AnomalyDetectorApiKey
             }
         }
 
+        private bool showAgeAndGender = false;
+        public bool ShowAgeAndGender
```

Comment:
We need to update VisionAPIExplorer and RecognitionPage to consume this as well.

Reply:
These demos use the FaceIdentificationBorder control to show age/gender data where I already added the toggle logic.